### ISSUE TITLE
ISIC splits

### ISSUE THREAD

**Issue by hmchuong** [chenwydj/ultra_high_resolution_segmentation#33]
Dear authors,

I appreciate your work in making the best method for high-resolution semantic segmentation. However, I would like to get the train/val/test splits that you used in experiments with ISIC dataset. 

Thank you,

**Comment by chenwydj** [comment]
Hi @hmchuong !

Thank you very much for your interest in our work!

Here are the splits: [train](https://drive.google.com/file/d/13e7Iz8k0eLI5qZCxNz_hXwQzWdRAwiVJ/view?usp=sharing), [validation](https://drive.google.com/file/d/1Pejljmsyc3s84Fo3_vPQQnzOZZXYRtLS/view?usp=sharing), [test](https://drive.google.com/file/d/1wxVO_yW0dlgXQVAC2CMWG7DIT1SvG4kp/view?usp=sharing).

Hope that helps!
